src/conn/conn_reconfig.c

```diff
@@ -496,6 +496,7 @@ __wt_conn_reconfig(WT_SESSION_IMPL *session, const char **cfg)
 	WT_ERR(__wt_sweep_config(session, cfg));
 	WT_ERR(__wt_verbose_config(session, cfg));
 	WT_ERR(__wt_timing_stress_config(session, cfg));
+	WT_ERR(__wt_las_config(session, cfg));
```

Comment:
Unless this needs to be done exactly here (in that case add a comment about why), move it to be alphabetized with the other calls (and fix `wt_timing_stress_config` to also be alphabetized).

Reply:
ca144d1